android/native.gradle

```diff
@@ -1,5 +1,5 @@
 project.ext.instabug = [
-    version: '12.7.1'
+    version: '13.0.0'
```

Comment:
the latest Android version for dream11 is 13.0.1

Reply:
I changed it, thanks 🙏🏼